docs/building.md

```diff
@@ -15,6 +21,12 @@ python3 -m venv ~/.venv/kuksa-client
 source ~/.venv/kuksa-client/bin/activate  # Run this every time you want to activate kuksa-client's virtual environment
 ```
 
+To use the right api interfaces of databroker run the following:
+```console
+python3 -m proto
```

Comment:
Not related to this PR, but I am thinking if we should add

`proto/**/*.py`

to the .gitignore of kuksa-databroker so that the submodule isn't reported as dirty after this command